services/user.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str, password: str, **kwargs) -> None:
+    default_kwargs = {"email": "", "first_name": "", "last_name": ""}
+    kwargs = {**default_kwargs, **kwargs}
+
+    get_user_model().objects.create_user(
+        username=username, password=password, **kwargs
+    )
```

Comment:
`README` has a list of parameters that should be allowed in this function

Reply:
Sorry, but I don't understand what my mistake is
I created the function according to this tutorial, it has 2 required and 3 optional parameters:
"create_user, takes username, password, optional parameters email, first_name, last_name. The method should save user properly (password should be encrypted), use method .create_user() for this purpose. Also, set fields if appropriate optional values are provided. "